src/api/overlays/Overlay.js

```diff
@@ -42,6 +43,10 @@ class Overlay extends EventEmitter {
         if (onDestroy) {
             this.once('destroy', onDestroy);
         }
+
+        if (onDismiss) {
```

Comment:
What's the difference between `dismiss` and `destroy`? I am concerned that overlay-specific details (like the distinction between `save` and `cancel`) are sneaking into the generic Overlay (which may be purely informational and not have any buttons at all).

Reply:
I think I might have fallen into the above mentioned trap of intended use of FormsAPI vs OverlayAPI.
`destroy` seems to be called by the Overlay as well as by the FormsAPI. 
`dismiss` was an attempt to keep the dismiss functionality within the Overlay as changing `destroy` could have ripple effects. Ideally, the FormsAPI would not be dictating when the Overlay dismisses - which is what is happening now.